### PR TITLE
628 e2e delete

### LINKED ISSUE #628: Make deleted record e2e test more robust
Check that the child deleted was the right one

### PR DESCRIPTION
**Pull request by mmazanec22**
Yay they all pass!

Closes #628

### CODE REVIEW

**Comment by jlhogan** [comment]
:'''''D

File: .circleci/config.yml [hunk 1]
```diff
@@ -117,10 +117,6 @@ jobs:
           working_directory: e2e-tests
           name: Install e2e test dependencies
           command: yarn
-      - run:
-          working_directory: e2e-tests
-          name: Download example data
```
Comment (jlhogan) [inline .circleci/config.yml:122]: why not this anymore?

File: e2e-tests/tests/deleteChild.js [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
+        "//*/button[contains(., 'Yes, delete record')]"
+      );
+      // TODO: change if we change alert header level
+      await browser.waitForElementVisible(
+        'xpath',
+        "//*/h2[contains(., 'Record deleted')]"
+      );
+      await browser.verify.containsText(
+        { locateStrategy: 'css selector', selector: '.usa-alert' },
+        lastName
```
Comment (bamader) [inline e2e-tests/tests/deleteChild.js:33]: Nice! Way cleaner than the selector I would've tried.

File: .circleci/config.yml (hunk 1 above)
Comment (mmazanec22) [reply .circleci/config.yml:122]: I don't think we need it anymore since the upload function now downloads the example file to upload

Comment (bamader) [reply .circleci/config.yml:122]: Looks like the functionality has been moved into an `uploadFile` util that downloads a file for any test that needs it. Probably makes the test suites more robust / independent that way?

Comment (mmazanec22) [reply .circleci/config.yml:122]: (I didn't implement that-- @bamader was that you?  Is that a correct understanding?)

**Review by bamader — APPROVED** [review]
This is the most beautiful thing I've ever seen 😭

File: .circleci/config.yml (hunk 1 above)
Comment (bamader) [reply .circleci/config.yml:122]: Oh lol wait I did do that 😂 . Sorry. Yes, you're right though that we don't need it anymore. I moved the code over but didn't delete the stuff in the `config` file.

Comment (bamader) [reply .circleci/config.yml:122]: I just got too caught up in the row of green checks 😍

Comment (jlhogan) [reply .circleci/config.yml:122]: awesome yes makes sense!!! i didn't know we could upload a file from the local-to-test machine

**Review by jlhogan — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (mmazanec22) [reply .circleci/config.yml:122]: I didn't think it was possible!  Huge kudos to @bamader for figuring that one out

Comment (jlhogan) [reply .circleci/config.yml:122]: woooooooooooooooOOO!!! goooooooo @bamader :partying_face:

**Pull request merged**